SwanFileBrowser/src/swanfilebrowser.ts

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) SWAN Development Team.
+// Author: Omar.Zapata@cern.ch 2021
+
+/**
+ * This file containts the implementation for SwanFileBrowser and SwanFileBrowserModel classes.
+ *
+ * Those classes are the replacement for the default FileBrowser and allows to
+ * manipulated the paths to check if the folder is a project or not and to set the proper kernel path.
+ */
```

Comment:
I believe the code in this file is mostly yours? If that is the case, it would be good to explicitly say it here in the header docs, for future reference.